.travis.yml

```diff
@@ -1,28 +1,53 @@
-language: node_js
+# although this is a node.js project, specify the use of java in order for OTP
+# and datatools-server to run as expected on java 8 specifically
+dist: trusty # jdk 8 not available on xenial
+language: java
```

Comment:
Is changing the language to java the only way to specify `oraclejdk8`? I'd rather the language be node here, but it's not really a huge deal in the grand scheme of things.

Reply:
There's probably another way to manually install it, but I opted for this because I thought it might result in more expedient Travis builds since the images might be a little more optimized for Java.

Reply:
Clever thought. Maybe just augment the comment with that thinking.